src/commonTest/kotlin/fr/acinq/lightning/channel/states/OfflineTestsCommon.kt

```diff
@@ -662,8 +662,11 @@ class OfflineTestsCommon : LightningTestSuite() {
         val commitTx = alice1.commitments.latest.localCommit.publishableTxs.commitTx.tx
         val (alice2, actions2) = alice1.process(ChannelCommand.Close.ForceClose)
         assertIs<Closing>(alice2.state)
-        actions2.hasPublishTx(commitTx)
-        assertNull(actions2.findOutgoingMessageOpt<Error>()) // we're offline so we shouldn't try to send messages
+        assertTrue(actions2.contains(ChannelAction.Storage.StoreState(alice2.state)))
+        assertEquals(commitTx, actions2.hasPublishTx(ChannelAction.Blockchain.PublishTx.Type.CommitTx))
```

Comment:
Isn't `actions2.hasPublishTx(commitTx)` better?

Reply:
Done in f6e4cec3.